### PR TITLE
Change Windows Priveleges and Check Privileges

### PR DESCRIPTION
**Pull request by retiutut**
* Update priveleges for Windows users and check if GUI has been run as Administrator

### CODE REVIEW

**Comment by retiutut** [comment]
@Andrey1994 I created a new account on a Windows 10 computer that was not an Administrator, then ran the GUI from this build. It didn't seem to trigger:

```
    if (!AdministratorChecker.IS_RUNNING_AS_ADMINISTRATOR) {
        println("OpenBCI_GUI: This application is not being run with Administrator access. This could limit the ability to connect to devices or read/write files.");
    }
```

File: OpenBCI_GUI/Extras.pde [hunk 1]
```diff
@@ -42,6 +48,47 @@ private void checkIs64BitJava() {
     }
 }
 
+//https://stackoverflow.com/a/23538961
+public static class AdministratorChecker
```
Comment (retiutut) [inline OpenBCI_GUI/Extras.pde:52]: Here is supposedly a method that works on more than just Windows, though it didn't seem to be triggered when running the app as a non-administrator and regular read/write access on Windows 10.

Comment (Andrey1994) [reply OpenBCI_GUI/Extras.pde:52]: I've found smth that looks reasonable: https://github.com/izpack/izpack/blob/82182fbafec83d1c5fd411dfefe57f9b09ad59c4/izpack-util/src/main/java/com/izforge/izpack/util/PrivilegedRunner.java 

lets copypaste method `public boolean isElevationNeeded(String path)` and `public boolean isAdminUser()` and print notification if any of these two returned `false`. For now only windows is needed

Comment (Andrey1994) [reply OpenBCI_GUI/Extras.pde:52]: actually these methods looks like they may work on unix too

**Pull request merged**